docs/configuration.rst

```diff
@@ -501,10 +501,10 @@ The following environment variables for the tracer are supported:
      version_added:
         v1.9.0:
 
-   DD_CIVISIBILITY_AGENTLESS_ENABLED:
+   DD_CIVISIBILITY_ENABLED:
      type: Boolean
      default: False
-     description: Configures the ``CIVisibility`` service to use an agent-less test reporting ``CIVisibilityWriter``.
+     description: Configures the ``CIVisibility`` service to use a test-reporting ``CIVisibilityWriter``.
```

Comment:
I think we should clarify either (or both) here or in the PR description what `test-reporting CIVisibilityWriter` means exactly for the user. My understanding so far is that the `CIVisibilityWriter` will send test events to the agentless endpoint by default, or to the agent via EVP proxy if the agent supports it. If `DD_CIVISIBILITY_ENABLED` is false, then we continue reporting test traces directly to the agent. Is that correct?